osu.Game/Overlays/Mods/ModSelectPanel.cs

```diff
@@ -192,10 +197,17 @@ private void playStateChangeSamples()
             if (samplePlaybackDisabled.Value)
                 return;
 
+            double? lastPlaybackTime = Statics.Get<double?>(Static.LastModSelectPanelSamplePlaybackTime);
```

Comment:
Please, when implementing something which almost precisely matches an existing usage (`HoverSampleDebounceComponent`), at very least copy the existing implementation rather than doing it a completely different way from scratch.

Specifically:

- using `.Get` is generally avoided as it has a slight overhead.
- Calling the variable `Statics` (and making it unnecessarily `protected`)
- using `is-not-null` instead of `!=null`

I'll fix it this time